e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java

```diff
@@ -518,6 +540,81 @@ public void addSupportForAutoscalingFilesystemForAWS() {
+                .enableClusterLaunch()
+                .clusterSettingsForm("Auto-scaled cluster")
+                .setWorkingNodesCount("1")
+                .ok()
+                .launchTool(this, nameWithoutGroup(tool));//
+        final String parentRunId = getLastRunId();
+        runsMenu()
+                .showLog(getLastRunId())
+                .waitForSshLink()
+                .ssh(shell -> {
+                    int num = shell
+                            .waitUntilTextAppears(parentRunId)
+                            .execute(command[0])
+                            .assertNextStringIsVisible("Grid engine profile has been found: main.q",
+                                    format("pipeline-%s", parentRunId))
+                            .execute(format(command[1], testProfile))
+                            .assertPageContains(format("Grid engine %s profile.", testProfile))
+                            .execute(":q!")
+                            .waitUntilTextAppears(parentRunId)
+                            .assertPageAfterCommandContainsStrings(command[1],
+                                    format("Grid engine %s profile has been created.", testProfile))
+                            .execute(command[0])
+                            .assertNextStringIsVisible(format("Grid engine profile has been found: %s", testProfile),
+                                    format("pipeline-%s", parentRunId))
+                            .execute(command[2])
+                            .assertPageContains(format("Grid engine %s profile.", testProfile))
+                            .getRowsNumber();
+                    shell
+                            .execute(format("%sdd", num))
+                            .inputText(command[3])
+                            .execute(":wq")
+                            .waitUntilTextAppears(parentRunId)
+                            .assertPageContainsString("Boolean parameter CP_CAP_AUTOSCALE has invalid value nontrue. Please specify true/false/yes/no/on/off.")
+                            .assertPageContainsString("Grid engine profile verification has failed. Reverting the changes...")
+                            .execute(command[2])
+                            .assertPageContains(format("Grid engine %s profile.", testProfile))
+                            .execute(format("%sdd", num))
+                            .inputText(command[4])
+                            .execute(":wq")
+                            .waitUntilTextAppears(parentRunId)
+                            .execute(format("qsub -b y -q %s -pe local 4 -t 1:2 sleep 5m", testProfile))
+                            .close();
+                });
+
+        Set<String> logMessages = runsMenu()
+                .showLog(parentRunId)
+                .clickTaskWithName("SGEProfiles")
+                .logMessages()
+                .collect(toSet());
+        Arrays.stream(expectedLogMessages())
+                .map(String::trim)
```

Comment:
I suppose we can avoid using `map(String::trim)` by performing the trimming inside the forEach method, isn't it?

Reply:
Done